help-beta/src/styles/main.css

```diff
@@ -3,3 +3,37 @@
     pointer-events: none;
     cursor: default;
 }
+
+.sl-markdown-content {
+    img {
+        vertical-align: top;
+        box-shadow: 0 0 4px hsl(0deg 0% 0% / 5%);
+        border: 1px solid hsl(0deg 0% 87%);
+        border-radius: 4px;
+        margin-top: 0;
+
+        &.emoji-small {
+            display: inline-block;
+            width: 1.25em;
+            box-shadow: none;
+            border: none;
+            vertical-align: text-top;
+        }
+
+        &.emoji-big {
+            display: inline-block;
+            width: 1.5em;
+            box-shadow: none;
+            border: none;
+            vertical-align: text-top;
+        }
+
+        &.help-center-icon {
+            display: inline-block;
+            width: 1.25em;
+            box-shadow: none;
+            border: none;
+            vertical-align: text-top;
+        }
```

Comment:
All these 3 inline image classes only differ on font-size, so it might be possible to pull the common properties out into another class. But maybe that is an exercise best left after the cutover, so as to not modify the current help docs css.